processing/src/main/java/io/druid/query/BaseQuery.java

```diff
@@ -145,10 +145,12 @@ public boolean getContextBoolean(String key, boolean defaultValue)
     return QueryContexts.parseBoolean(this, key, defaultValue);
   }
 
-  protected Map<String, Object> computeOverridenContext(Map<String, Object> overrides)
+  protected static Map<String, Object> computeOverriddenContext(
```

Comment:
This PR breaks compatibility of BaseQuery by refactoring this protected instance method into a static one. Is it a compatibility bug?